qa/integration-tests/src/test/java/io/camunda/it/utils/MultiDbConfigurator.java

```diff
@@ -26,30 +23,36 @@ public class MultiDbConfigurator {
 
   private final TestStandaloneApplication<?> testApplication;
 
-  private final OperateProperties operateProperties;
-  private final TasklistProperties tasklistProperties;
-
   public MultiDbConfigurator(final TestStandaloneApplication<?> testApplication) {
     this.testApplication = testApplication;
-    // we are configuring this always, even if we might not use the applications
-    operateProperties = new OperateProperties();
-    tasklistProperties = new TasklistProperties();
-
-    testApplication
-        .withBean("operate-config", operateProperties, OperateProperties.class)
-        .withBean("tasklist-config", tasklistProperties, TasklistProperties.class);
   }
 
   public void configureElasticsearchSupport(
       final String elasticsearchUrl, final String indexPrefix) {
-    operateProperties.getElasticsearch().setUrl(elasticsearchUrl);
-    operateProperties.getElasticsearch().setIndexPrefix(indexPrefix);
-    operateProperties.getZeebeElasticsearch().setUrl(elasticsearchUrl);
-    operateProperties.getZeebeElasticsearch().setPrefix(indexPrefix);
-    tasklistProperties.getElasticsearch().setUrl(elasticsearchUrl);
-    tasklistProperties.getElasticsearch().setIndexPrefix(indexPrefix);
-    tasklistProperties.getZeebeElasticsearch().setUrl(elasticsearchUrl);
-    tasklistProperties.getZeebeElasticsearch().setPrefix(indexPrefix);
+
+    final Map<String, Object> elasticsearchProperties = new HashMap<>();
+
+    /* Tasklist */
+    elasticsearchProperties.put("camunda.tasklist.elasticsearch.url", elasticsearchUrl);
+    elasticsearchProperties.put("camunda.tasklist.zeebeElasticsearch.url", elasticsearchUrl);
+    elasticsearchProperties.put("camunda.tasklist.elasticsearch.index-prefix", indexPrefix);
+    elasticsearchProperties.put("camunda.tasklist.zeebeElasticsearch.prefix", indexPrefix);
+
+    /* Operate */
+    elasticsearchProperties.put("camunda.operate.elasticsearch.url", elasticsearchUrl);
+    elasticsearchProperties.put("camunda.operate.zeebeElasticsearch.url", elasticsearchUrl);
+    elasticsearchProperties.put("camunda.operate.elasticsearch.index-prefix", indexPrefix);
+    elasticsearchProperties.put("camunda.operate.zeebeElasticsearch.prefix", indexPrefix);
+
+    /* Camunda */
+    elasticsearchProperties.put(
+        "camunda.database.type",
+        io.camunda.search.connect.configuration.DatabaseType.ELASTICSEARCH);
+    elasticsearchProperties.put("camunda.database.indexPrefix", indexPrefix);
+    elasticsearchProperties.put("camunda.database.url", elasticsearchUrl);
+
+    testApplication.withAdditionalProperties(elasticsearchProperties);
+
```

Comment:
I remember there was a need for using the properties and setting the beans. But I might be wrong. Let's see what the CI says.